src/helpers/label.ts

```diff
@@ -1,8 +1,11 @@
-import { getBotContext, getLogger } from "../bindings";
+import { Context } from "probot";
+import { getBotConfig, getBotContext, getLogger } from "../bindings";
 import { COLORS } from "../configs";
-import { Payload } from "../types";
+import { calculateBountyPrice } from "../handlers";
+import { Label, Payload } from "../types";
+import { deleteLabel } from "./issue";
 
-export const listLabelsForRepo = async (per_page?: number, page?: number): Promise<string[]> => {
+export const listLabelsForRepo = async (per_page?: number, page?: number, all?: boolean): Promise<Label[] | string[]> => {
```

Comment:
could we make it returning only `Label[]` like 
```suggestion
export const listLabelsForRepo = async (per_page?: number, page?: number): Promise<Label[]> => {
```
There would be a few parts to use `string[]` but we can get them easily from `Label[]`. we can reduce the complexity in use of the `listLabelsForRepo` function.